watch.py

```diff
@@ -0,0 +1,133 @@
+import datetime
+import time
+
+def clear():
+  import os
+  os.system("CLS")
+
+dictionary = {
+"0":
+    """
+    ■■■■■
+    ■   ■
+    ■   ■
+    ■   ■
+    ■■■■■""",
+"1":
+    """
+      ■  
+     ■■  
+    ■ ■  
+      ■  
+    ■■■■■""",
+"2":
+    """
+    ■■■■■
+    ■   ■
+      ■  
+    ■    
+    ■■■■■""",
+"3":
+    """
+    ■■■■ 
+        ■
+      ■  
+        ■
+    ■■■■ """,
+"4":
+    """
+    ■   ■
+    ■   ■
+    ■ ■ ■
+        ■
+        ■""",
+"5":
+    """
+    ■■■■■
+    ■    
+    ■ ■ ■
+        ■
+    ■■■■■""",
+"6":
+    """
+    ■■■■■
+    ■    
+    ■ ■ ■
+    ■   ■
+    ■■■■■""",
+"7":
+    """
+    ■■■■■
+        ■
+      ■  
+      ■  
+      ■  """,
+"8":
+    """
+    ■■■■■
+    ■   ■
+    ■■■■■
+    ■   ■
+    ■■■■■""",
+"9":
+    """
+    ■■■■■
+    ■   ■
+    ■ ■ ■
+        ■
+    ■■■■■""",
+":":
+    """
+         
+      ■  
+         
+      ■  
+         """,
+" ":
+    """
+         
+         
+         
+         
+         """
+
+}
+
+while True:
+    now = datetime.datetime.now()
+    timestr = now.strftime("%H:%M:%S")
+
+    a = dictionary[timestr[0]]
+    b = dictionary[timestr[1]]
```

Comment:
names like a, b, c and so on - bad idea...
The reviewer must decipher the meaning of the variables